pages/create/index.page.js

```diff
@@ -151,6 +153,8 @@ const Create = () => {
           name={END_DATE}
           onChange={(event) => handleDate(END_DATE, event.target.value)}
           type="date"
+          min={getMaxMinDate(getValues(START_DATE), 1)}
```

Comment:
Why 1 and 20? is this the min and max interval? should be env variables. This is configured on the contract when deployed 

Please take a look at the crowdfunding contracts.